handsontable/src/editors/autocompleteEditor/autocompleteEditor.js

```diff
@@ -484,82 +484,84 @@ export class AutocompleteEditor extends HandsontableEditor {
 
     super.onBeforeKeyDown(event);
   }
-}
 
-/**
- * Filters and sorts by relevance.
- *
- * @param {*} value The selected value.
- * @param {string[]} choices The list of available choices.
- * @param {boolean} caseSensitive Indicates if it's sorted by case.
- * @returns {number[]} Array of indexes in original choices array.
- */
-AutocompleteEditor.sortByRelevance = function(value, choices, caseSensitive) {
-  const choicesRelevance = [];
-  const result = [];
-  const valueLength = value.length;
-  let choicesCount = choices.length;
-  let charsLeft;
-  let currentItem;
-  let i;
-  let valueIndex;
-
-  if (valueLength === 0) {
-    for (i = 0; i < choicesCount; i++) {
-      result.push(i);
+  /**
+   * Filters and sorts by relevance.
+   *
+   * @param {*} value The selected value.
+   * @param {string[]} choices The list of available choices.
+   * @param {boolean} caseSensitive Indicates if it's sorted by case.
+   * @returns {number[]} Array of indexes in original choices array.
+   */
+  sortByRelevance = function(value, choices, caseSensitive) {
```

Comment:
Maybe we can change the current test spec for that to check `locale` with `sortByRelevance`?

Reply:
Please look at: https://github.com/handsontable/handsontable/pull/8897#discussion_r742127406